### PR TITLE
Fix Windows CustomData

### PR DESCRIPTION
**Pull request by anhowe**
This fixes the windows customdata where it was not taking some of the public keys as parameters.

### CODE REVIEW

File: parts/kuberneteswindowssetup.ps1 [hunk 1]
```diff
@@ -32,8 +32,8 @@ param(
     $AzureHostname
 )
 
-$global:CACertificate = "<<<caCertificate>>>"
```
Comment (colemickens) [inline parts/kuberneteswindowssetup.ps1]: I like this and the simplification of the template function below!

File: parts/kubernetesmastervars.t [hunk 2]
```diff
@@ -155,7 +155,7 @@
     "agentWindowsSku": "2016-Datacenter-with-Containers",
     "agentWindowsVersion": "2016.0.20170127",
     "singleQuote": "'",
-    "windowsCustomScriptSuffix": " $inputFile = '%SYSTEMDRIVE%\\AzureData\\CustomData.bin' ; $outputFile = '%SYSTEMDRIVE%\\AzureData\\CustomDataSetupScript.ps1' ; $inputStream = New-Object System.IO.FileStream $inputFile, ([IO.FileMode]::Open), ([IO.FileAccess]::Read), ([IO.FileShare]::Read) ; $sr = New-Object System.IO.StreamReader(New-Object System.IO.Compression.GZipStream($inputStream, [System.IO.Compression.CompressionMode]::Decompress)) ; $sr.ReadToEnd() | Out-File($outputFile) ; Invoke-Expression('{0} {1}' -f $outputFile, $arguments) ; "
+    "windowsCustomScriptSuffix": " $inputFile = '%SYSTEMDRIVE%\\AzureData\\CustomData.bin' ; $outputFile = '%SYSTEMDRIVE%\\AzureData\\CustomDataSetupScript.ps1' ; Copy-Item $inputFile $outputFile ; Invoke-Expression('{0} {1}' -f $outputFile, $arguments) ; "
```
Comment (colemickens) [inline parts/kubernetesmastervars.t]: Can you summarize this? The new line is much, much shorter. What all changed?

Comment (colemickens) [inline parts/kubernetesmastervars.t]: Where is $arguments populated?

Comment (raveeram) [inline parts/kubernetesmastervars.t]: I think $arguments is populated in the agent extensions resource template where the script is invoked.

Comment (anhowe) [inline parts/kubernetesmastervars.t]: we can no longer zip the file.  This is part of the full script run in CSE, and $arguments is used there

Comment (colemickens) [inline parts/kubernetesmastervars.t]: I see... this gets appended inside the CSE where the `$arguments` is formed... Huh. Ok.
